src/tests/data_handlers/user_data_handler.test.js

```diff
@@ -0,0 +1,280 @@
+import UserDataHandler from '../../data_handlers/user_data_handler';
+import testData from '../../../data/users.json';
+import serchParamsTestData from '../../../data/isMatchingAllSearchParamsTestData.json';
+import findUsersTestData from '../../../data/findUsersTestData.json';
+describe('user_data_handler test suite', () => {
+  let userDataHadlerObject;
+  beforeEach(() => {
+    userDataHadlerObject = new UserDataHandler();
+  });
+  describe('getNumberOfUsers() method check', () => {
+    test('getNumberOfUsers() returns 10 if users were loaded', async () => {
```

Comment:
it's better to use should statement, as we can't assume that the result would be the same as we claimed.